### PR TITLE
Jg shelter review us2

### PR DESCRIPTION
**Pull request by elguapogordo**
- created reviews table
  - created foreign key for shelters
- added test for
  - reviews creation
  - add reviews to shelter show page
- added reviews creation functionality
- added reviews to shelter show page

### CODE REVIEW

File: app/views/shelters/show.html.erb [hunk 1]
```diff
@@ -7,4 +7,14 @@
   <div class="shelter_options"><%= link_to "Update Shelter", "/shelters/#{@shelter.id}/edit" %></div>
   <div class="shelter_options"><%= link_to "Delete Shelter", "/shelters/#{@shelter.id}", :method => :delete %></div>
   <div class="shelter_options"><%= link_to "See Our Pets", "/shelters/#{@shelter.id}/pets" %></div>
+
+  <section class="review-section">
+    <% @shelter.reviews.each do |review| %>
+      <%= image_tag ( review.review_image ) %>
```
Comment (elguapogordo) [inline app/views/shelters/show.html.erb]: was initially html tag. changed to activerecord helper

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Review < ApplicationRecord
+
```
Comment (mikez321) [inline app/models/review.rb]: If we're going to format things like this we should go through and make sure it is all the same... just FYI

File: app/models/shelter.rb [hunk 3]
```diff
@@ -1,5 +1,8 @@
 class Shelter < ApplicationRecord
+
```
Comment (mikez321) [inline app/models/shelter.rb]: Already fixing some of the spacing, like I had made above.  Thanks!

File: app/views/shelters/show.html.erb [hunk 4]
```diff
@@ -7,4 +7,14 @@
   <div class="shelter_options"><%= link_to "Update Shelter", "/shelters/#{@shelter.id}/edit" %></div>
   <div class="shelter_options"><%= link_to "Delete Shelter", "/shelters/#{@shelter.id}", :method => :delete %></div>
   <div class="shelter_options"><%= link_to "See Our Pets", "/shelters/#{@shelter.id}/pets" %></div>
+
+  <section class="review-section">
```
Comment (mikez321) [inline app/views/shelters/show.html.erb]: Heads up that we've made this class, but we haven't styled it.  Let's make a note on that in the project board.

File: spec/features/shelterreviews/shelter_review_index_spec.rb [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                               state: "MA",
+                               zip: "01001")
+
+    shelter2 = Shelter.create(name: "The Cat Palace",
+                               address: "998 Kings Ct",
+                               city: "Kings Landing",
+                               state: "CA",
+                               zip: "90210")
+
+    review1 = shelter1.reviews.create!(review_image: "https://via.placeholder.com/150",
```
Comment (mikez321) [inline spec/features/shelterreviews/shelter_review_index_spec.rb]: Also to make a note of: Tony wanted us to make sure this was formatted properly.  Right now it isn't looking the way he was talking about...

**Comment by mikez321** [comment]
Looks good, we'll need to touch up a few formatting things we talked about but good work on this!

**Pull request merged**